src/uct/base/uct_iface.h

```diff
@@ -323,6 +323,13 @@ typedef struct uct_iface_mpool_config {
     }
 
 
+#define UCT_TL_IFACE_PUT_DESC(_desc) \
+    do { \
```

Comment:
we do not use `do while(0)`, because of coding style

Reply:
fixed